src/sass/patterns/_buttons.scss

```diff
@@ -0,0 +1,11 @@
+// Buttons
+// =======
+
+// Mimic SLDS :disabled styles
+.btn-disabled,
+.btn-disabled:hover {
+  cursor: default;
+  background-color: $color-background-alt;
+  color: $color-text-button-default-disabled;
+  border-color: $color-border;
+}
```

Comment:
@stacyk @mirisuzanne I added this as a workaround to fix tooltips on disabled buttons.